### DISCUSSION TITLE
Empowering devs with AI: How Shopify made GitHub Copilot core to its culture - [AI2857M] [Day 1 Universe]

### DISCUSSION THREAD

**Discussion by ghostinhershell** [community/community#74747] [Copilot Conversations]
**Speaker:** @fnthawar (Farhan Thawar, Shopify’s VP and Head of Engineering)
**Time:** Wednesday, Nov 8, 2:45 PM - 3:25 PM PST

> Join Farhan Thawar, Shopify’s VP and Head of Engineering, as he discusses how Shopify, an e-commerce platform that builds at incredible scale, is unlocking the potential of its developers with AI tooling like GitHub Copilot. Learn strategies to increase developer empowerment with the adoption of AI tooling, measure the success of your GitHub Copilot adoption, and prepare for the downstream effects of increased developer productivity.

Thank you for joining the session! If you have any questions or feedback regarding what was presented, please feel free to add them in the comments and I'll/we'll do our best to answer.

- To watch the recording, please click [`here`](https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog/session/1696016249677001G4GQ).👈
- [Presentation Resources](https://static.rainfocus.com/github/universe23/sess/1696016249677001G4GQ/sessionppt/AI2857M_Empowering%20devs%20with%20AI%20How%20Shopify%20made%20GitHub%20Copilot%20core%20to%20its%20culture_1699551111192001K4hy.pdf)

**Comment by jricciardi** [comment]
Tons of Copilot usage! Super cool story from Farhan

![Screenshot 2023-11-08 at 3 07 58 PM](https://github.com/community/community/assets/31521880/24c96080-414e-4fa3-af23-1a8bc8ff5a3b)

**Reply by jricciardi** [reply]
More cool stats 👀 

![Screenshot 2023-11-08 at 3 08 59 PM](https://github.com/community/community/assets/31521880/eee2a002-fe8d-4951-8936-17ac346edc9f)

**Comment by ghostinhershell** [comment]
Love hearing about how Copilot is breaking barriers for teams at Shopify! Starting a project in a language you're unfamiliar with is daunting for sure so love to hear that Copilot reduces the barrier to entry for teams at Shopify 🚀

**Reply by gitstua** [reply]
The Shopify engineering team are producing some amazing things. They have some of their open source contributions right here on GitHub [github.com/Shopify](https://github.com/Shopify)

**Comment by Fla12fonsecas** [comment]
It was amazing! I liked so much these points:
- the reason why developers choose not to use copilot.
- the code error copilot create to generate a code to say "Hello world"
- the qualitative metric is much better than the quantitative for this case
- the allowance to use of copilot during the interviews
- the reality to now know if the numbers of usage is low or high, because it is a process, and should be progressive
- the difference between the gain expected and the real gains generated
- 
By the way, I am from Brazil. If there is someone from Brazil also, lets chat :)

**Reply by mishmanners** [reply]
Hi to you from Brazil. I'm tuning in from Australia ❤️
